composer.json

```diff
@@ -15,30 +15,29 @@
     ],
     "config" : {
         "platform": {
-            "php": "5.3.9"
+            "php": "7.2.5"
         }
     },
     "bin": [
         "bin/phpbrew"
     ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": "^5.3.9||^7.0||^8.0",
-        "corneltek/cliframework": "3.0.x-dev#2650a53433854d43b91955e8f967c62ce07869d7",
+        "php": "^7.2.5||^8.0",
```

Comment:
note that by doing this you are also allowing PHP 8.2, 8.3 & co. which may not be true

Reply:
You're right. Maybe it can change into: `^7.2.5||<=8.1`.